packages/react-composites/tests/browser/call/app/index.tsx

```diff
@@ -19,6 +19,8 @@ import { IDS } from '../../common/constants';
 import { isMobile, verifyParamExists } from '../../common/testAppUtils';
 import memoizeOne from 'memoize-one';
 import { IContextualMenuItem, mergeStyles } from '@fluentui/react';
+import { fromFlatCommunicationIdentifier } from '@internal/acs-ui-common';
+import { CommunicationUserIdentifier } from '@azure/communication-signaling';
```

Comment:
This should be imported from `@azure/communication-common`, not signaling.

Reply:
Yeah, changed in a follow up PR: https://github.com/Azure/communication-ui-library/pull/1039/files#diff-c662184caec88e95fe7a065186766d62d6e684b9dfb12bf6248708e748638f31R4